test/suites/dev-tanssi/register-with-relay-proof/test_deregister_with_relay_proof.ts

```diff
@@ -160,19 +159,11 @@ describeSuite({
                 expect(balanceBeforeAlice.reserved.toBigInt()).to.be.eq(expectedDepositValue);
                 expect(balanceAfterAlice.reserved.toBigInt()).to.be.eq(0n);
 
-                // Find a Deposit(100000000000000) event for bob address
                 const events = await polkadotJs.query.system.events();
-                const filtered = filterAndApply(events, "balances", ["Deposit"], ({ event }: EventRecord) =>
-                    (event.data as unknown as { amount: u128 }).toJSON()
+                const fee = extractFeeAuthor(events, bob.address).amount.toBigInt();
+                expect(balanceAfterBob.free.toBigInt()).toEqual(
+                    balanceBeforeBob.free.toBigInt() + expectedDepositValue - fee
                 );
-                const bobDepositEvent = filtered.find(
-                    (event) => event[0] === bob.address && BigInt(event[1]) === expectedDepositValue
-                );
-                if (!bobDepositEvent) {
-                    console.log("deposit events: ", filtered);
-                }
-                expect(bobDepositEvent).to.not.be.undefined;
```

Comment:
why remove this? isnt there an event?

Reply:
There isn't an event anymore, unfortunately (at least not that I could find)

Reply:
can we at least check that Balances::Holds contains the amount held?